docs/integration/windows.md

```diff
@@ -77,3 +77,58 @@ Moreover, a set of pre-defined [alerts](https://sematext.com/docs/alerts/) is su
 - Network List Service (NlaSvc)
 
 In addition to having alerts for these services, you can [create alerts](https://sematext.com/docs/alerts/creating-metrics-alerts/) for any other Windows Service you like, as long as it is available on your Windows machines.
+
+## Logs
+
+Starting from [version 3.7.0](https://sematext.com/docs/agents/sematext-agent/releasenotes/#version-370), [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) is capable of collecting Windows Event logs along with metrics related with the status of Windows services.
+
+### Install Sematext Agent
+
+1. Create a Windows Logs [App](https://sematext.com/docs/guide/app-guide/). This will let you install the agent and control access to your logs data.
+2. Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) according to the instructions displayed in the UI
```

Comment:
Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) according ... ==> Install [Sematext Agent for Windows](https://sematext.com/docs/agents/sematext-agent/windows-installation/) on each Windows host you want to monitor according .... ?